doc/install_and_upgrade.md

```diff
@@ -194,13 +204,6 @@ If you use the
 [ArchHaskell repository](https://wiki.archlinux.org/index.php/ArchHaskell), you
 can also get the `haskell-stack-tool` package from there.
 
-There is also an unofficial package in the Arch community repository.  Note that this Stack
-version may lag behind, so we recommend running `stack upgrade` after installing
```

Comment:
It doesn't seem necessary to me, the version in the repository is always kept up to date. And in the small chance this isn't' the case, and you run it once and forget about it, your local version will lag behind if you don't keep doing it.